### PR TITLE
fix: fix No version in the history version drawer if a file is duplicated : EXO-61044 

### PR DESCRIPTION
**Pull request by GouadriaHanen**
prior to this change, after duplicating a document, the history version drawer is empty since no version is added
after this change, an auto version is added after creating a document

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -67,7 +68,13 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
-
+      <dependency>
```
Comment (ahamdi) [inline pom.xml:71]: Could you import the parent POM of ECMS instead ?

File: documents-storage-jcr/src/test/java/org/exoplatform/documents/storage/jcr/JCRDocumentFileStorageTest.java [hunk 2]
```diff
@@ -185,8 +189,11 @@ public void duplicateDocument() throws Exception {
     when(currentNode.addNode("copy of test","nt:file")).thenReturn(currentNode);
     when(identity.getRemoteId()).thenReturn("username");
     when(JCRDocumentsUtil.getUserSessionProvider(repositoryService, userID)).thenReturn(sessionProvider);
+    when(CommonsUtils.getService(AutoVersionService.class)).thenReturn(autoVersionService);
+    when(autoVersionService.autoVersion(any(Node.class))).thenReturn(null);
```
Comment (ahamdi) [inline documents-storage-jcr/src/test/java/org/exoplatform/documents/storage/jcr/JCRDocumentFileStorageTest.java]: This instruction not needed IMO, you will just spy on the autoVersionService to check if the function was called, it won't be called really.

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (GouadriaHanen) [reply pom.xml:71]: done!

File: documents-storage-jcr/src/test/java/org/exoplatform/documents/storage/jcr/JCRDocumentFileStorageTest.java (hunk 2 above)
Comment (GouadriaHanen) [reply documents-storage-jcr/src/test/java/org/exoplatform/documents/storage/jcr/JCRDocumentFileStorageTest.java]: done

**Review by ahamdi — APPROVED** [review]

File: pom.xml [hunk 3]
```diff
@@ -67,7 +68,13 @@
...
+        <groupId>org.exoplatform.ecms</groupId>
+        <artifactId>ecms</artifactId>
+        <version>${org.exoplatform.ecms.version}</version>
```
Comment (hbenali) [inline pom.xml:74]: Wrong property name Use `addon.exo.ecms.version` instead. cc @ahamdi

**Comment by hakermi** [comment]
Ecms dependancy is not allowed in documents app

**Pull request merged**